src/content/docs/guides/JSON/0-json_intro.mdx

```diff
@@ -89,6 +92,39 @@ This command creates sub-folders for each type of resource. One of these is name
   ```
 
   </TabItem>
+  <TabItem label="Object-Oriented">
+
+  ```csharp
+using SplashKitSDK;
+
+namespace GameDataManagement
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Define the file path
```

Comment:
I would remove these comments in the OOP version as there is no comments in the C++ or the C# top level statement version. So the comments within this OOP one look really out of place.